pkg/platform/runtime/buildexpression/buildexpression.go

```diff
@@ -592,10 +592,6 @@ func (e *BuildExpression) getSolveNode() (*Ap, error) {
 			continue
 		}
 
-		if a.Assignment.Name == "" || a.Assignment.Name != "runtime" {
```

Comment:
I removed this code because it was causing a failure with this build script: https://github.com/ActiveState/cli/blob/878706429d6b1aa137258db5955faf8741bb7f23/pkg/platform/runtime/buildscript/buildscript_test.go#L80-L104

Reply:
Does the function now just return the first solve node it finds? You are correct that we don't want `runtime = solve()` I believe it's there because we need a way to identify _one_ solve node that we can make modifications to. I recall this was how PB suggested we do it, but that could be incorrect or may have changed. Would you be able to follow up with them?

Reply:
After talking to PB, multiple solve nodes are not supported, and will not be supported until the VCS migration is complete. I've reverted this change, disabled the failing test, and filed https://activestatef.atlassian.net/browse/DX-2238 to re-enable that test.